engine/src/elections.rs

```diff
@@ -176,6 +176,9 @@ where
 
 						pending_submissions.insert(election_identifier,	(partial_vote, vote));
 					},
+					Ok(None) => {
+						info!("Voting task for election '{:?}' returned 'None' (nothing to submit).", election_identifier);
```

Comment:
nit: I feel like debug would be more appropriate here otherwise we'll see a lot of spam.

Reply:
```suggestion
						debug!("Voting task for election '{:?}' returned 'None' (nothing to submit).", election_identifier);
```